src/devices_models/devices/TwoTerminalDC_branches.jl

```diff
@@ -418,6 +418,47 @@ function add_constraints!(
     return
 end
 
+function add_constraints!(
```

Comment:
Why are we doing this? If its copper plate it should be no-opt

Reply:
But we model HVDC lines in different subnetworks right? Copperplate balance for each subnetwork using the ref buses, so we want to model HVDC lines between different subnetworks.